types/svg2png/index.d.ts

```diff
@@ -1,10 +1,18 @@
 // Type definitions for svg2png node package 
 // Project: https://github.com/domenic/svg2png
-// Definitions by: hans windhoff <https://github.com/hansrwindhoff>
+// Definitions by: songChengcheng <https://github.com/sccgithub>
```

Comment:
Add yourself to the end of the list: `// Definitions by: hans windhoff <https://github.com/hansrwindhoff>, songChengcheng <https://github.com/sccgithub>`